PDBminer

```diff
@@ -132,6 +149,9 @@ def check_input_file(df):
             cluster_id = [999] * len(df)
             df['cluster_id'] = cluster_id
     
+    if len(hugo_name_list) == len(df):
```

Comment:
what is this is false, what would the reason be? This check doesn't look very fail-proof (i.e. if for some reason the list is shorter of just 1 element, we don't add anything at all)

Reply:
This is a leftover i think. the list hugo_name_list is always empty, since nothing is ever added to it, so I removed it entirely.